### PR TITLE
DevEx Release Notes Updates

### PR DESCRIPTION
**Pull request by bmcelvee**
Additional entries needed.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@codyhoag - just a heads up, I'm adding the rest of the DevEx release notes content. One of the links is giving me a bit of trouble (might pull it).

**Comment by bmcelvee** [comment]
@gabemontero and @dmage this PR is adding the final release notes for DevEx in 4.5 - would you mind giving it a quick review, please?

**Comment by dmage** [comment]
LGTM

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review, thanks!

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -402,6 +395,17 @@ affinity scheduler must be manually configured.
...
 
+[id="ocp-4-5-virtual-hosted-buckets-s3"]
+==== Virtual hosted buckets for custom S3 endpoints
+
+Virtual hosted buckets are now supported to deploy clusters in new or hidden AWS regions.
+
+[id="ocp-4-5-node-pull-secrets"]
+==== Node pull credentials during build and imagestream import
+
+Builds and Imagestream imports will automatically use the pull secret used to install the cluster if a pull secret is not explicitly set. Developers will not need to copy this pull secret into their namespace.
```
Comment (bobfuru) [inline release_notes/ocp-4-5-release-notes.adoc:406]: Do you want to match capitalization for "build" and "imagestream" between the h4 and first sentence?

**Review by bobfuru — COMMENTED** [review]
Couple of suggestions and otherwise LGTM!

File: release_notes/ocp-4-5-release-notes.adoc (hunk 1 above)
Comment (bobfuru) [reply release_notes/ocp-4-5-release-notes.adoc:406]: Instead of "will not" in the second sentence, how about "do not"? Just a preference.

**Pull request merged**